src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -701,7 +701,7 @@ void Convolution::setPostOps(dnnl::primitive_attr& attr,
                         continue;
                     }
                     DEBUG_LOG(getName(), ": Append ", node->getName(), " as legacy post op");
-                    eltwiseNode->appendPostOps(ops, dims, args);
+                    eltwiseNode->appendPostOps(ops, dims, args, 1);
```

Comment:
better to have a named constant instead of a magic number, i.e. channelAxis
applicable to all the `appendPostOps `changes

Reply:
Done